### PR TITLE
Edit header and navbar

### PR DESCRIPTION
**Pull request by aikot1125**
This is my first take on the header and the navbar. 
I have two JavaScript questions which are commented in the sidemenu.js file. It would be great if one of you can help me with them!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,719 +23,758 @@
...
     <link rel="stylesheet" href="css/jquery.timepicker.css">
 
-    
+
     <link rel="stylesheet" href="css/flaticon.css">
     <link rel="stylesheet" href="css/icomoon.css">
     <link rel="stylesheet" href="css/style.css">
-  </head>
-  <body>
+    <script src="https://kit.fontawesome.com/b98fb790ca.js" crossorigin="anonymous"></script>
```
Comment (sbuckingham89) [inline index.html:30]: Love the font! For branding purposes, I think we should chat in slack about the 2-3 fonts we want to consistently use throughout the page and we can create CSS variables for them.  We should also probably decide on our color scheme too!

File: index.html [hunk 2]
```diff
@@ -22,719 +23,758 @@
...
+                    </li>
+                    <li class="nav-item"><a href="#" class="nav-link">About</a></li>
+                    <li class="nav-item"><a href="blog.html" class="nav-link">Blog</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+                    <li class="nav-item cta cta-colored"><a href="#" class="nav-link"><span class="icon-shopping_cart"></span>[0]</a></li>
+
+                </ul>
+            </div>
+        </div>
+    </nav> -->
```
Comment (sbuckingham89) [inline index.html]: I think we can probably take the original navbar out - unless you had it commented out to refer back to at some point?

File: index.html [hunk 3]
```diff
@@ -22,719 +23,758 @@
...
+                    </a>
+                </div>
+                <div class="col-md-7 py-5 wrap-about pb-md-5 ftco-animate">
+                    <div class="heading-section-bold mb-5 mt-md-5">
+                        <div class="ml-md-0">
+                            <h2 class="mb-4">Modist <br>Online <br> <span>Fashion Shop</span></h2>
+                        </div>
+                    </div>
+                    <div class="pb-md-5">
+                        <p>On her way she met a copy. The copy warned the Little Blind Text, that where it came from it would have been rewritten a thousand times and everything that was left from its origin would be the word "and" and the Little Blind Text
```
Comment (sbuckingham89) [inline index.html:266]: Was this an intended break in code here? If you have Prettier, or some sort of linter, it's probably why these other bigger chunks of code are different than before (just inserting indentions and what not).

File: js/sidemenu.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+let openSideMenu = document.querySelector('.fa-bars');
```
Comment (sbuckingham89) [inline js/sidemenu.js:2]: `querySelector` only selects the first one named `.fa-bars` - just want to confirm that we won't reuse that class somewhere else.

File: js/sidemenu.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+openSideMenu.addEventListener('click', function() {
+    document.getElementById('side-menu').style.width = '100%';
+    // When clicking the hamburger icon for the first time, the side menu appears abruptly. From the second time onwards it shows up from the side. Is there a way to make the menu show up from the side from the first click?
```
Comment (sbuckingham89) [inline js/sidemenu.js]: There is more than likely a way to make it work that way! It's hard for me to visualize what it's doing until I fire it up in my local.  Want to video chat sometime this week and we can look at it together?

File: js/sidemenu.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+    document.getElementById('side-menu').style.display = 'block';
+});
+
+// I want to click on the hamburger icon to close the sidemenu instead of the x button.. Do you know an easy way to do this?
```
Comment (sbuckingham89) [inline js/sidemenu.js]: Yes! We could discuss this if we meet too!

**Review by sbuckingham89 — DISMISSED** [review]
Left a few comments! Looks great!!

File: index.html (hunk 2 above)
Comment (aikot1125) [reply index.html]: I'll remove it in the next PR!

File: index.html (hunk 3 above)
Comment (aikot1125) [reply index.html:266]: Not sure what happened here! Just now I tried removing the indentation but when I hit save it reverts back. Perhaps it's one of my extensions.. I don't know why it does that.

File: js/sidemenu.js (hunk 6 above)
Comment (aikot1125) [reply js/sidemenu.js]: 👍

**Review by sbuckingham89 — APPROVED** [review]
These look good to me!

**Pull request merged**